src/perspective-mercator-viewport.js

```diff
@@ -252,6 +252,22 @@ export default class WebMercatorViewport extends Viewport {
       [lng + deltaLng, lat + deltaLat, Z + deltaZ];
   }
 
+  /**
+   * Returns map settings {latitude, longitude, zoom}
```

Comment:
The doc header is inconsistent, should probably say returns a new WebMercatorViewport

If we only support 2D case, we should mention that.